### ISSUE TITLE
Reporobot not active

### ISSUE THREAD

**Issue by GammaBurst101** [jlord/git-it-electron#218]
In one of the tutorials on Git-it, it asks us to add @reporobot as a member to the project. I did that. 
Then I was asked to wait and after that, pull from the remote.
I waited for 2 months but every time I do "git pull origin add-GammaBurst101", it says "already up-to-date".

What should I do?

**Comment by diogoburgosr** [comment]
I'm having the same issue. I did everything as @nicknish and clicked VERIFY. It's verifying for too long.
I think my problem is I'm behind a corporate proxy. 
Calling @jlord ! (does that even works? First time using github.)

**Comment by JohnnyLAmpAz** [comment]
Yeah, same thing here but Git-It marked it as complete due to `Up to date!` status.
